### ISSUE TITLE
Attestation of key update

### ISSUE THREAD

**Issue by tomt1664** [commerceblock/lockbox#53]
Lockbox to provide attestation of `statechain_id` key update (providing proof of key share inaccessibility). The attestation should be provided in the key-update finalize return object (`KUAttest`). 

```
KUFinalize { 
    statechain_id: Uuid,
    shared_key_id: Uuid,
}

KUAttest {
    statechain_id: Uuid,
    attestation: String,
}
```
This attestation will consist of an SGX quote, which is a cryptographic measurement of the instantiated enclave. The quote is signed using the attestation collateral that’s stored in the caching service (following the DCAP).

https://www.intel.com/content/www/us/en/developer/articles/technical/quote-verification-attestation-with-intel-sgx-dcap.html
